root/user/UserProfile.js

```diff
@@ -860,14 +861,15 @@ const UserProfile = ({
   addedEntities,
 }: UserProfileProps): React.Element<typeof UserAccountLayout> => {
   const viewingOwnProfile = !!$c.user && $c.user.id === user.id;
+  const adminViewing = Boolean($c.user) && isAccountAdmin($c.user);
```

Comment:
Only commenting since I noticed the previous lines uses `!!` already. :)  `!= null` might be clearer than either since that's really what we're guarding against?
```suggestion
  const viewingOwnProfile = $c.user != null && $c.user.id === user.id;
  const adminViewing = $c.user != null && isAccountAdmin($c.user);
```